app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala

```diff
@@ -397,4 +401,21 @@ class DesConnector @Inject()(
         }
     }
   }
+  def parseConflictDesResponse(res:UpstreamErrorResponse): DesResponse ={
```

Comment:
Hmm some of this code is redundant. I see you're trying to mimic the behaviour of the parseDesResponse function but by taking in an UpstreamErrorResponse, it means a lot of this functionality doesn't apply. e.g. you wouldn't get a 2xx status code as an error response.